Dockerfile

```diff
@@ -9,9 +9,9 @@ RUN apt-get update && apt-get install -y gcc python3-dev
 # Install the required dependencies via pip
 COPY pyproject.toml pyproject.toml
 COPY poetry.lock poetry.lock
-RUN pip install "poetry==1.8.3"
-RUN poetry config virtualenvs.create false
-RUN poetry install
+RUN pip install "poetry==1.8.3" \
+    && poetry config virtualenvs.create false \
+    && poetry install --only main
```

Comment:
`--without dev` would be probably more self explanatory but we use `--only main` in other projects so we can keep it as it is.

Reply:
I was getting a deprecation warning when I was testing this unfortunately 
```
The `--no-dev` option is deprecated, use the `--only main` notation instead.
```
I agree though, it reads better to me too 😢

Reply:
Ah sorry, I misread your original comment 🤦 I've tried with `--without dev` and it still works (without deprecation warning), so will change to that now, thanks

Reply:
I think that `only main` just installs main dependencies, whereas `without dev` installs _everything_ except dev dependencies. 

So it might be safer to keep it as `only main` for now, just in case at some point we end up adding another dependency group?